[{"authorTime":"2017-10-05 07:38:18","codes":[{"authorDate":"2017-10-05 07:38:18","commitOrder":1,"curCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","date":"2017-10-05 11:18:56","endLine":879,"groupId":"10737","id":1,"instanceNumber":1,"isCurCommit":0,"methodName":"portClashOfPrimaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/ca/779a44c49daf64ead469014da81e6f86fb0528.src","preCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":864,"status":"B"},{"authorDate":"2017-10-05 07:38:18","commitOrder":1,"curCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(SocketUtils.findAvailableTcpPort(40000));\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","date":"2017-10-05 11:18:56","endLine":898,"groupId":"2083","id":2,"instanceNumber":2,"isCurCommit":0,"methodName":"portClashOfSecondaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/ca/779a44c49daf64ead469014da81e6f86fb0528.src","preCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(SocketUtils.findAvailableTcpPort(40000));\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":882,"status":"B"}],"commitId":"33edfdab46085422e04b171cd2834e996bfe2cb9","commitMessage":"@@@Merge branch 'gh-9316'\n","date":"2017-10-05 11:18:56","modifiedFileCount":"0","status":"B","submitter":"Phillip Webb"},{"authorTime":"2018-04-05 18:22:35","codes":[{"authorDate":"2017-10-05 07:38:18","commitOrder":2,"curCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","date":"2017-10-05 11:18:56","endLine":879,"groupId":"10737","id":3,"instanceNumber":1,"isCurCommit":0,"methodName":"portClashOfPrimaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/ca/779a44c49daf64ead469014da81e6f86fb0528.src","preCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":864,"status":"N"},{"authorDate":"2018-04-05 18:22:35","commitOrder":2,"curCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","date":"2018-04-05 18:35:58","endLine":946,"groupId":"2083","id":4,"instanceNumber":2,"isCurCommit":0,"methodName":"portClashOfSecondaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/1d/aa28f4d5b87efce0118f25ddb95daa52b546ba.src","preCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(SocketUtils.findAvailableTcpPort(40000));\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":931,"status":"M"}],"commitId":"c3bf8541d75b1beef767f896e9326ab1241e813e","commitMessage":"@@@Do not rely on findAvailableTcpPort finding an available port\n\nCloses gh-12779\n","date":"2018-04-05 18:35:58","modifiedFileCount":"2","status":"M","submitter":"Andy Wilkinson"},{"authorTime":"2019-02-04 20:36:33","codes":[{"authorDate":"2019-02-04 20:36:33","commitOrder":3,"curCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","date":"2019-02-04 20:36:33","endLine":922,"groupId":"15725","id":5,"instanceNumber":1,"isCurCommit":0,"methodName":"portClashOfPrimaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/be/dd89db0851f4434362a430fdd395c8d21d582c.src","preCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":911,"status":"M"},{"authorDate":"2019-02-04 20:36:33","commitOrder":3,"curCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","date":"2019-02-04 20:36:33","endLine":936,"groupId":"15765","id":6,"instanceNumber":2,"isCurCommit":0,"methodName":"portClashOfSecondaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/be/dd89db0851f4434362a430fdd395c8d21d582c.src","preCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException()\n\t\t\tthrows IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\ttry {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory\n\t\t\t\t\t\t.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t\tfail();\n\t\t\t}\n\t\t\tcatch (RuntimeException ex) {\n\t\t\t\thandleExceptionCausedByBlockedPort(ex, port);\n\t\t\t}\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":925,"status":"M"}],"commitId":"45cc649c7db6c5f205f512878956890078d2ecbf","commitMessage":"@@@Merge branch '2.1.x'\n","date":"2019-02-04 20:36:33","modifiedFileCount":"26","status":"M","submitter":"Andy Wilkinson"},{"authorTime":"2019-05-24 18:24:29","codes":[{"authorDate":"2019-05-24 18:24:29","commitOrder":4,"curCode":"\tprotected void portClashOfPrimaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","date":"2019-06-07 19:25:54","endLine":836,"groupId":"15725","id":7,"instanceNumber":1,"isCurCommit":0,"methodName":"portClashOfPrimaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/3a/d43aa3647fe43b229f82a62bb32039d0be8f53.src","preCode":"\tpublic void portClashOfPrimaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":827,"status":"M"},{"authorDate":"2019-05-24 18:24:29","commitOrder":4,"curCode":"\tvoid portClashOfSecondaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","date":"2019-06-07 19:25:54","endLine":848,"groupId":"15765","id":8,"instanceNumber":2,"isCurCommit":0,"methodName":"portClashOfSecondaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/3a/d43aa3647fe43b229f82a62bb32039d0be8f53.src","preCode":"\tpublic void portClashOfSecondaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":839,"status":"M"}],"commitId":"b18fffaf14f9ce3e5651f44745019890e8a899c2","commitMessage":"@@@Move tests to JUnit 5 wherever possible\n","date":"2019-06-07 19:25:54","modifiedFileCount":"1282","status":"M","submitter":"Andy Wilkinson"},{"authorTime":"2019-06-14 06:11:18","codes":[{"authorDate":"2019-06-14 06:11:18","commitOrder":5,"curCode":"\tprotected void portClashOfPrimaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPortOnPrimaryConnector(ex, port));\n\t\t});\n\t}\n","date":"2019-06-14 06:11:18","endLine":836,"groupId":"15725","id":9,"instanceNumber":1,"isCurCommit":0,"methodName":"portClashOfPrimaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/1c/bb13ce32047e18ecdfe755e88400efa852d42a.src","preCode":"\tprotected void portClashOfPrimaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":827,"status":"M"},{"authorDate":"2019-06-14 06:11:18","commitOrder":5,"curCode":"\tvoid portClashOfSecondaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPortOnSecondaryConnector(ex, port));\n\t\t});\n\t}\n","date":"2019-06-14 06:11:18","endLine":848,"groupId":"15765","id":10,"instanceNumber":2,"isCurCommit":0,"methodName":"portClashOfSecondaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/1c/bb13ce32047e18ecdfe755e88400efa852d42a.src","preCode":"\tvoid portClashOfSecondaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPort(ex, port));\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":839,"status":"M"}],"commitId":"4c7e457582c00aae351e2ad9a97b495c46a71dda","commitMessage":"@@@Improve analysis of tomcat bind exception\n\nUsing the throwOnFailure attribute on the tomcat connector.  we can now\ndetermine if the underlying exception was a BindException and throw\na PortInUseException instead of the generic WebServerException.\n\nCloses gh-7130\n","date":"2019-06-14 06:11:18","modifiedFileCount":"8","status":"M","submitter":"Madhura Bhave"},{"authorTime":"2019-12-20 23:28:40","codes":[{"authorDate":"2019-12-20 23:28:40","commitOrder":6,"curCode":"\tprotected void portClashOfPrimaryConnectorResultsInPortInUseException() throws Exception {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPortOnPrimaryConnector(ex, port));\n\t\t});\n\t}\n","date":"2019-12-20 23:28:40","endLine":839,"groupId":"102004","id":11,"instanceNumber":1,"isCurCommit":0,"methodName":"portClashOfPrimaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/ba/b8246d9357645b595347d992afa88a8588c68f.src","preCode":"\tprotected void portClashOfPrimaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\tfactory.setPort(port);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPortOnPrimaryConnector(ex, port));\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":830,"status":"M"},{"authorDate":"2019-12-20 23:28:40","commitOrder":6,"curCode":"\tvoid portClashOfSecondaryConnectorResultsInPortInUseException() throws Exception {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPortOnSecondaryConnector(ex, port));\n\t\t});\n\t}\n","date":"2019-12-20 23:28:40","endLine":851,"groupId":"102004","id":12,"instanceNumber":2,"isCurCommit":0,"methodName":"portClashOfSecondaryConnectorResultsInPortInUseException","params":"()","path":"/mnt/clonedata/CloneManagementServer/ManagementServer/consistResult/result-springboot-10-0.7/blobInfo/CC_OUT/blobs/ba/b8246d9357645b595347d992afa88a8588c68f.src","preCode":"\tvoid portClashOfSecondaryConnectorResultsInPortInUseException() throws IOException {\n\t\tdoWithBlockedPort((port) -> {\n\t\t\tassertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> {\n\t\t\t\tAbstractServletWebServerFactory factory = getFactory();\n\t\t\t\taddConnector(port, factory);\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer = factory.getWebServer();\n\t\t\t\tAbstractServletWebServerFactoryTests.this.webServer.start();\n\t\t\t}).satisfies((ex) -> handleExceptionCausedByBlockedPortOnSecondaryConnector(ex, port));\n\t\t});\n\t}\n","realPath":"spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/servlet/server/AbstractServletWebServerFactoryTests.java","repoName":"springboot","snippetEndLine":0,"snippetStartLine":0,"startLine":842,"status":"M"}],"commitId":"07857f7f24232f52a108d4f30e93a3aefd43814c","commitMessage":"@@@Merge branch '2.2.x'\n\nCloses gh-19418\n","date":"2019-12-20 23:28:40","modifiedFileCount":"3","status":"M","submitter":"Andy Wilkinson"}]
